wp/resources/sample_binaries/debruijn/main_1.c

```diff
@@ -0,0 +1,107 @@
+#include <assert.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+// taken from http://supertech.csail.mit.edu/papers/debruijn.pdf
```

Comment:
Would it be worth having a small sentence summarizing this pdf? The paper itself is somewhat involved for anyone trying to understand the problem.